src/gui/util_gui.cpp

```diff
@@ -348,18 +348,22 @@ namespace Util {
 	
 	void Marker::drawCenterMarker(QPainter* painter, const QPointF& center)
 	{
-		const auto saved_hints = painter->renderHints();
-		painter->setRenderHint(QPainter::Antialiasing, true);
+		painter->save();
+		
+		const auto larger_radius = mmToPixelPhysical(1.1);
+		const auto smaller_radius = larger_radius*3/4;
 
-		painter->setPen(Qt::white);
+		painter->setRenderHint(QPainter::Antialiasing, Settings::getInstance().getSettingCached(Settings::MapDisplay_Antialiasing).toBool());
+		auto pen = QPen(Qt::white);
+		pen.setWidth(larger_radius - smaller_radius);
```

Comment:
`QPen::setWidth` takes an `int` but the actual argument is a `qreal`. Either we need to call `QPen::setWidthF`, or round explicitly.

Reply:
This is already flagged with a warning in Qt Creator.

Reply:
Interesting that my GCC 9 reported dozens of other warnings on current `origin/dev` but not this particular one. Which compiler/code analyzer complained about this line?

On related note, what are the plans towards static code analyzers and code changes arising from that kind of automated review? On QuickEvent Coverity [did quite good job](https://github.com/fvacek/quickbox/commits?author=lpechacek).